scripts/common.js

```diff
@@ -0,0 +1,18 @@
+import { Client, Intents } from 'discord.js';
+import { config } from 'dotenv';
+
+export const get$1 = () => {

```

Comment:
Nitpick: I think it'd be better to just name this `getNArg(n)` and use it as
`getNArg(1)` than naming it `$1`.

Reply:
Cool idea. Resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/190/commits/14e74d093af1c9acac0da1f31d4946dea6e5925c>.